### PR TITLE
Gemm+Bilinear

### PR DESCRIPTION
**Pull request by asroy**
* Use ```DeviceGemmMultipleD``` to implement GEMM+Bilinear: ```E=alpha*A*B+beta*D```
* Remove ```DeviceGemmBias```, ```DeviceGemmBiasActivation```, ```DeviceGemmBiasActivationAdd``` and other device op derived from them

### CODE REVIEW

**Review by shaojiewang — APPROVED** [review]

File: library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_km_kn_mn_instance.cpp [hunk 1]
```diff
@@ -14,9 +16,9 @@ namespace tensor_operation {
 namespace device {
 namespace instance {
 
-using F16     = ck::half_t;
-using F32     = float;
-using F16_F16 = ck::Tuple<F16, F16>;
+using F16           = ck::half_t;
+using F32           = float;
+using F16_F16_Tuple = ck::Tuple<F16, F16>;
```
Comment (asroy) [inline library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_km_kn_mn_instance.cpp:21]: F16_F16_TUPLE to be consistent

File: library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_km_nk_mn_instance.cpp [hunk 2]
```diff
@@ -14,9 +16,9 @@ namespace tensor_operation {
 namespace device {
 namespace instance {
 
-using F16     = ck::half_t;
-using F32     = float;
-using F16_F16 = ck::Tuple<F16, F16>;
+using F16           = ck::half_t;
+using F32           = float;
+using F16_F16_Tuple = ck::Tuple<F16, F16>;
```
Comment (asroy) [inline library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_km_nk_mn_instance.cpp:21]: F16_F16_TUPLE to be consistent

File: library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_mk_kn_mn_instance.cpp [hunk 3]
```diff
@@ -14,9 +16,9 @@ namespace tensor_operation {
 namespace device {
 namespace instance {
 
-using F16     = ck::half_t;
-using F32     = float;
-using F16_F16 = ck::Tuple<F16, F16>;
+using F16           = ck::half_t;
+using F32           = float;
+using F16_F16_Tuple = ck::Tuple<F16, F16>;
```
Comment (asroy) [inline library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_mk_kn_mn_instance.cpp:21]: F16_F16_TUPLE to be consistent

File: library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_mk_nk_mn_instance.cpp [hunk 4]
```diff
@@ -14,9 +16,9 @@ namespace tensor_operation {
 namespace device {
 namespace instance {
 
-using F16     = ck::half_t;
-using F32     = float;
-using F16_F16 = ck::Tuple<F16, F16>;
+using F16           = ck::half_t;
+using F32           = float;
+using F16_F16_Tuple = ck::Tuple<F16, F16>;
```
Comment (asroy) [inline library/src/tensor_operation_instance/gpu/gemm_add_add_fastgelu/device_gemm_add_add_fastgelu_xdl_c_shuffle_f16_f16_f16_mk_nk_mn_instance.cpp:21]: F16_F16_TUPLE to be consistent

**Pull request merged**